pyproject.toml

```diff
@@ -11,10 +11,12 @@ python = "^3.10"
 Django = "^4.2.2"
 pytest = "^7.3.2"
```

Comment:
Pytest and pytest django should be listed under 
```
[tool.poetry.group.dev.dependencies]
```
See https://python-poetry.org/docs/managing-dependencies/